addons/binding/org.openhab.binding.max/ESH-INF/thing/bridge.xml

```diff
@@ -34,6 +34,12 @@
 				<description>Binding settings for this device</description>
 				<advanced>false</advanced>
 			</parameter-group>
+			<!-- Trigger actions from habmin -->
+            <parameter-group name="actions">
```

Comment:
you now have tabs/spaces mixed in this file. Could you reapply the formatter here?